.github/workflows/sonarcloud.yml

```diff
@@ -128,9 +129,9 @@ jobs:
           cd $GITHUB_WORKSPACE/_build
           ctest -C Release --output-on-failure -L "unit"
 
-      - name: Compile coverage reports
+      - name: Collect coverage into one XML report
         run: |
-          cmake --build $GITHUB_WORKSPACE/_build --target code-coverage
+          gcovr --sonarqube > coverage.xml
```

Comment:
From the doc
```suggestion
          gcovr --sonarqube --output coverage.xml
```

Reply:
The example for cfamily from the docs suggested the > but we can try the option and if it doesn't smooth the paths wrongly

Reply:
From `man gcovr`
```
       -o OUTPUT, --output OUTPUT
              Print output to this filename. Defaults to stdout.  Required for
              --html-details.
```
So redirection from stdout to `coverage.xml` and this option should be equivalent.